sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url_async.py

```diff
@@ -98,7 +98,7 @@ async def test_content_url_pdf(self, client):
         self.assertEqual(layout.page_number, 1)
         self.assertFormPagesHasValues(result)
         self.assertEqual(layout.tables[0].row_count, 3)
-        self.assertEqual(layout.tables[0].column_count, 6)
+        self.assertEqual(layout.tables[0].column_count, 5)
```

Comment:
Why did this change?

Reply:
Improvements in the layout model. It was finding the wrong amount of columns previously.